plugins/MultiSites/Dashboard.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\MultiSites;
+
+use Piwik\API\ResponseBuilder;
+use Piwik\Config;
+use Piwik\Metrics\Formatter;
+use Piwik\Period;
+use Piwik\DataTable;
+use Piwik\DataTable\Row;
+use Piwik\DataTable\Row\DataTableSummaryRow;
+use Piwik\Plugins\API\ProcessedReport;
+use Piwik\Site;
+use Piwik\View;
+
+/**
+ * Fetches and formats the response of `MultiSites.getAll` in a way that it can be used by the All Websites AngularJS
+ * widget. Eg sites are moved into groups if one is assigned, stats are calculated for groups, etc.
+ */
+class Dashboard
```

Comment:
The class doc says nothing about sorting/offset/limit (or in other words supporting pagination), yet that appears to be the main reason to use Dashboard.php and not MultiSites.getAll directly. It's important in quickly understanding what the class is for.